service_restaurant/src/main/kotlin/org/thechance/service_restaurant/data/collection/RestaurantCollection.kt

```diff
@@ -24,6 +25,9 @@ data class RestaurantCollection(
     val id: ObjectId = ObjectId()
     val isDeleted: Boolean = false
 
+    val addressIds: List<@Contextual Id<AddressCollection>> = emptyList()
```

Comment:
why type AddressCollection, not ObjectId or Id<String>?

Reply:
It's already Id for AddressCollection